virtualizarr/writers/icechunk.py

```diff
@@ -242,21 +243,17 @@ def write_virtual_variable_to_icechunk(
         )
     else:
         append_axis = None
-
-        # Get the codecs and convert them to zarr v3 format
-        codecs = zarray._v3_codecs()
-
-        # create array if it doesn't already exist
+        # TODO: Should codecs be an argument to zarr's AsyncrGroup.create_array?
```

Comment:
You're asking for an upstream change here right?

Reply:
I think so, should I create an issue for that?

Reply:
If you think it would be useful then go for it. Shouldn't stop us merging this though.

Reply:
https://github.com/zarr-developers/zarr-python/issues/2848